app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PostsController < Api::V1::BaseController
+      def index
+        @user = User.find_by(id: params[:user_id])
```

Comment:
No need to define `@user` as a instance variable, as you're not using it in your json response